playbooks/common/openshift-cluster/upgrades/upgrade.yml

```diff
@@ -203,7 +215,7 @@
     command: >
       {{ openshift.common.admin_binary }} manage-node {{ openshift.common.hostname | lower }} --schedulable=true
     delegate_to: "{{ groups.oo_first_master.0 }}"
-    when: inventory_hostname in groups.oo_nodes_to_config and openshift.node.schedulable | bool
```

Comment:
Yeah that's probably the part I'm missing, I was watching master (although that's what the bug reports on), I think I can see how it would happen if you move a node to unschedulable. 

I'm genuinely not sure how masters are winding up schedulable in his example...

Still doing some testing.